app/App/Panel/Local/index.js

```diff
@@ -447,6 +447,19 @@ class Settings extends React.Component {
               When should Frame relock your hot signers?
             </div>
           </div>
+          <div className='signerPermission localSetting' style={{ zIndex: 198 }}>
+            <div className='signerPermissionControls'>
+              <div className='signerPermissionSetting'>Log reporting</div>
```

Comment:
"Exception reporting", or "Log exceptions".  Leaving towards the former, so my other comments reflect this.  @mholtzman which do you think?

Reply:
"Reporting" has a negative connotation I feel, "Log" sounds more friendly, and the description is clear that "logging" means to send the exceptions.